src/main/resources/META-INF/plugin.xml

```diff
@@ -58,6 +58,8 @@
                 instance="org.nixos.idea.settings.NixLangSettingsConfigurable"/>
 
         <formattingService implementation="org.nixos.idea.format.NixExternalFormatter"/>
+
+        <notificationGroup id="NixIDEA" displayType="BALLOON" key="notification.group.nixidea"/>
```

Comment:
issue: I think `key` is supposed to reference a value from a *“bundle”*, i.e. the files containing translations for different languages. We don't have such a file right now. I can try to find out how to handle this case.

Reply:
👍  let me know. For now I simply removed `id=` and the plugin seems to work

Reply:
Just removing the `key` attribute seems to be the right solution.